ddtrace/internal/remoteconfig/worker.py

```diff
@@ -92,7 +92,7 @@ def start_subscribers(self):
         log.debug("[%s][P: %s] Remote Config Poller fork. Starting Pubsub services", os.getpid(), os.getppid())
         self._client.renew_id()
         for pubsub in self._client.get_pubsubs():
-            pubsub.restart_subscriber()
+            pubsub.restart_subscriber(True)
```

Comment:
Since the old thread is not running there shouldn't be anything to join. Since we dropped support for Python 2 I don't think it is a problem here, but IIRC this used to be an issue with old interpreters. Just a note in case something weird surfaces.